webapp/src/app.tsx

```diff
@@ -251,6 +251,15 @@ export class ProjectView
             }
             else if (msg.type === "thumbnail") {
                 if (this.shareEditor) this.shareEditor.setThumbnailFrames((msg as pxsim.SimulatorAutomaticThumbnailMessage).frames);
+            } else if (msg.type === "multiplayer") {
+                // todo move over types from multiplayer app to pxsim/embed.ts
+                const multiplayerMessage = msg as any;
```

Comment:
(noting here that I'll be trying to move this over to `simulatorPresenceBar` without losing nice updates / clearing on restart in a followup pr)